### PR TITLE
 Add release notes for 0.279.2

### PR DESCRIPTION
**Pull request by NikhilCollooru**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.279.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 Release 0.279
 =============
 
+.. warning::
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/release/release-0.279.rst:5]: I don't think this needs a warning.  Adjusted queue size is a metric used for operations, it's not a correctness issue.

File: presto-docs/src/main/sphinx/release/release-0.279.2.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.279.2
+===============
+
+General Changes
```
Comment (jingmeih) [inline presto-docs/src/main/sphinx/release/release-0.279.2.rst:5]: Do we need to add `Upgrade Airlift to 0.207` as it's what https://github.com/prestodb/presto/commit/0325d3d4dffd4a251603544c6eed57e8453c7519 is doing.

**Review by jingmeih — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-0.279.2.rst [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+_______________
+* Fix a bug in calculating the adjusted queue size in ResourceGroupStates which could cause a larger adjusted queue size.
+* Upgrade Airlift to 0.207
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/release/release-0.279.2.rst:8]: I would actually remove this.  It's more relevant to developers, who can easily look at the git blame to see when this upgrade occurred, but not relevant to the audience for these release notes, users/operators of Presto.

**Review by tdcmeehan — APPROVED** [review]
LGTM % nit

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**